hawc/apps/summary/models.py

```diff
@@ -149,6 +153,65 @@ def print_node(node, depth):
                 print_node(node, 2)
 
 
+class SummaryTable(models.Model):
+    objects = managers.SummaryTableManager()
+
+    GENERIC = 0
```

Comment:
wanna try using the django3 enum choices?

https://docs.djangoproject.com/en/3.0/ref/models/fields/#field-choices-enum-types